Dockerfile.dapper

```diff
@@ -4,17 +4,21 @@ ARG DAPPER_HOST_ARCH
 ENV HOST_ARCH=${DAPPER_HOST_ARCH} ARCH=${DAPPER_HOST_ARCH} \
     DAPPER_ENV="REPO TAG QUAY_USERNAME QUAY_PASSWORD GITHUB_SHA MAKEFLAGS CLUSTERS_ARGS DEPLOY_ARGS E2E_ARGS RELEASE_ARGS" \
     DAPPER_SOURCE=/go/src/github.com/submariner-io/submariner-operator DAPPER_DOCKER_SOCKET=true \
-    OPERATOR_SDK_VERSION=0.12.0 \
-    CONTROLLER_GEN_VERSION=0.3.0 \
+    OPERATOR_SDK_VERSION=1.0.1 \
+    KUSTOMIZE_VERSION=3.5.4 KUSTOMIZE=$(GOBIN)/kustomize \
+    CONTROLLER_GEN_VERSION=0.3.0 CONTROLLER_GEN=$(GOBIN)/controller-gen \
```

Comment:
Defining `KUSTOMIZE` and `CONTROLLER_GEN` here without using them in the Dockerfile strikes me as odd.

Furthermore, we're not doing this "style" for any other command so why are these two special?

Finally, /go/bin is already a part of `$PATH` so you don't actually need to specify it.

Reply:
Thanks for the review @mkolesnik I will remove those envs and call the cli commands directly in `Makefile`